lib/Models/Catalog/Ows/WebMapServiceCatalogItem.ts

```diff
@@ -54,7 +55,23 @@ import proxyCatalogItemUrl from "../proxyCatalogItemUrl";
 import WebMapServiceCapabilities from "./WebMapServiceCapabilities";
 import WebMapServiceCapabilitiesStratum from "./WebMapServiceCapabilitiesStratum";
 import WebMapServiceCatalogGroup from "./WebMapServiceCatalogGroup";
-import TerriaFeature from "../../Feature/Feature";
+
+// Remove problematic query parameters from URLs (GetCapabilities, GetMap, ...) - these are handled separately
+const QUERY_PARAMETERS_TO_REMOVE = [
+  "request",
+  "service",
+  "x",
+  "y",
+  "width",
+  "height",
+  "bbox",
+  "layers",
+  "styles",
+  "version",
```

Comment:
'version' is a valid query parameter for a GetCapabilities request. Do we need to retain this somehow?
most WMS servers will default to the highest version they support anyway, and version is only reqlly critical in a GetMap request because it can dictate how you define your other query parameters... So can probably leave it out.